src/mantine-core/src/components/Box/Box.tsx

```diff
@@ -5,11 +5,9 @@ import {
   PolymorphicRef,
   extractSystemStyles,
 } from '@mantine/styles';
-import { useSx, BoxSx } from './use-sx/use-sx';
+import { useSx } from './use-sx/use-sx';
 
-interface _BoxProps extends Omit<DefaultProps, 'sx'> {
-  sx?: BoxSx;
-}
+interface _BoxProps extends DefaultProps {}
```

Comment:
Be aware that using an empty object will raise [@typescript-eslint/no-empty-interface](https://typescript-eslint.io/rules/no-empty-interface). Using directly DefaultProps instead of _BoxProps could fix it, but ignoring that warning is fine and makes more sense to keep consistency with others component.